templates/footer.twig

```diff
@@ -4,10 +4,7 @@
         {% include 'footer_country_selector.twig' %}
     </div>
     {% endif %}
-    <div class="container d-flex flex-column flex-lg-row-reverse">
-        {% if (social_overrides is defined and social_overrides|length) or footer_social_menu|length %}
-            {% include 'footer_social_media.twig' %}
-        {% endif %}
+    <div class="container d-flex flex-column flex-lg-row">
```

Comment:
I've removed the `*-reverse` class since is not compatible with a18y. Why? Because even if the element is re-arranged, that beaviour doesn't modify the `focus` tab. [The re-arrangement might be added programatically](https://github.com/greenpeace/planet4-master-theme/pull/2187/files#diff-7413612e8358cee4efb121d4eece663de1227d3f1caf111b09506a840a8a4dc3R10-R49).